OJP/OJP_FareSupport.xsd

```diff
@@ -231,9 +231,9 @@
 			<xs:documentation>[related to FARE PRODUCT in TM and NeTEx] different FARE PRODUCTs that may be available with related information</xs:documentation>
 		</xs:annotation>
 		<xs:sequence>
-			<xs:element name="FareProductId" type="FareProductCodeType">
+			<xs:element name="FareProductCode" type="FareProductCodeType">
```

Comment:
This renaming should probably be a separate pull request.
It will be forgotten in the release notes as breaking change otherwise.

Reply:
in separate